crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -458,10 +458,10 @@ mod tests {
 
     #[test]
     fn test_fixup_webpack_filename() {
-        let abs_path = "webpack:///../node_modules/@sentry/browser/esm/helpers.js";
+        let filename = "webpack:///../node_modules/@sentry/browser/esm/helpers.js";
```

Comment:
can we add another testcase, maybe with `webpack:///./app/utils/requestError/createRequestError.tsx` as example.